### ISSUE TITLE
FILE TRANSFER MODE - MacOS 

### ISSUE THREAD

**Issue by otternet** [polyend/TrackerIssues#522]
Hi Polyend, I’ve tried every app available for a Mac to use MTP, but somehow it just doesn’t connect. Maybe because I am using a Mac with USB-C only and using a dongle or hub, SyncMate, Commander One etc does not recognize PT when using the File Transfer mode. Some claim to have success with it, but I can’t seem to. Can you find a protocol that’s agnostic and allows Macs and WinTel machines to use this file transfer feature?

**Comment by jgrimsong** [comment]
I had same problem until I used Android File Transfer tool. That's the only one that worked for me.

**Comment by otternet** [comment]
> I had same problem until I used Android File Transfer tool. That's the only one that worked for me.

How did you connect it to your Mac? Can you share your steps? I downloaded the same, but it doesn't work for me. Are you running Big Sur? I've even tried OpenMTP but nothing seems to work.

**Comment by jgrimsong** [comment]
I just installed it on my iMac and have the USB cable attached to my hub. I'm running Catalina. Everything just came up automatically.

**Comment by otternet** [comment]
I'm using a dongle to my MacBook Pro running Big Sur but nothing shows up  🤷🏻‍♂️ Gonna try that again.

**Comment by otternet** [comment]
![image](https://user-images.githubusercontent.com/72027406/102287047-8e3c3a00-3f74-11eb-857c-b271e67f567c.jpeg)

Nope... grr! Still doesn't work. Maybe because I'm using the USB-C dongle to my MacBook.

**Comment by b0ll3** [comment]
Let's start with the most simple thing first:
is your device in File Transfer Mode?
Tracker's screen should look like in the picture.
![FTM](https://user-images.githubusercontent.com/73824590/102295903-9bddc980-3f4c-11eb-9610-e1c8ec80eefd.jpg)

**Comment by otternet** [comment]
Yup it is. I've also tried different "startup" sequences. Launch app first and then switch on FTM, and vice versa. Used a hub, used a dongle. The only thing I have not tried is plugging the included USB cable from Polyend to USB-A input, but my MacBook Pro only has USB-C. I have the "first generation" Tracker (flat keys), and from what I understand the second generation Trackers have a more flex USB-C to USB-C connection. Am wondering if that's the problem here.

**Comment by otternet** [comment]
UPDATE: no luck with direct to USB-A input sans hub or dongle either 🤷🏻‍♂️

Polyend... any suggestions from y'all?
